src/core/server/metrics/event_loop_delays/event_loop_delays_monitor.ts

```diff
@@ -6,10 +6,23 @@
  * Side Public License, v 1.
  */
 
+import moment from 'moment';
 import type { IntervalHistogram as PerfIntervalHistogram } from 'perf_hooks';
 import { monitorEventLoopDelay } from 'perf_hooks';
 import type { IntervalHistogram } from '../types';
 
+/**
+ * Nanosecond to milisecond conversion unit
+ */
+export const ONE_MILLISECOND_AS_NANOSECONDS = 1000000;
+
+/**
+ * Converts time metric from ns to ms
+ **/
+export function nsToMs(metric: number) {
+  return moment.duration(metric / ONE_MILLISECOND_AS_NANOSECONDS).asMilliseconds();
```

Comment:
I'm torn between using `moment` here and just treating the `number` as a `number`.

Reply:
Let's use the native API whenever possible. I'd remove `moment` usage as it doesn't bring any value.